staging/src/k8s.io/apiserver/pkg/authentication/user/user.go

```diff
@@ -70,8 +70,10 @@ func (i *DefaultInfo) GetExtra() map[string][]string {
 const (
 	SystemPrivilegedGroup = "system:masters"
 	NodesGroup            = "system:nodes"
-	AllUnauthenticated    = "system:unauthenticated"
-	AllAuthenticated      = "system:authenticated"
+	// TODO(logicalhan): configure flowcontrol for requests from this group
```

Comment:
can you describe what you envision for this? not sure this TODO belongs in a list of constants

Reply:
:P 

Yeah, I was thinking we probably don't actually want to give this role the same sort of priority as more critical requests. Philosophically, I believe that instrumentation should degrade gracefully without imploding the control-plane (i.e. I'd rather have my control-plane working without metrics than have my metrics take down my control-plane).

Reply:
There are a couple of monitoring/alerting philosophies at play here, I'm sure you can learn a lot more than from me internally at Google, but the gist is there are vaguely two opinions: when unable to monitor, you don't know the state of the system, so you assume the worst and read it as unavailability, or not. This is of course largely more about measuring, than the applications behavior, but I would keep this in mind. I tend to prefer the first, meaning monitoring should actually have equal if not higher priority than the rest (much like schedling priority itself), as otherwise we cannot actually reason about this system.

Reply:
Yeah, and I don't necessarily disagree with you but this is specifically about flow control and prioritization of traffic to the apiserver. Saying metrics endpoint traffic is lower priority doesn't mean that metrics are low priority. It just means if we get DDOS'd on the metrics endpoint then we'll start queuing some of those requests and letting stuff like internal apiserver self-requests go through (and hopefully succeed).

Reply:
thanks for the context, can you copy this thread into an issue rather than a todo here?

Reply:
I'm going to remove the TODO since someone has filed an issue around this already (https://github.com/kubernetes/kubernetes/issues/93359) and we are basically converging on reusing this group to fix that existing issue.